reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+starting a version of PowerShell, the `$Env:PSModulePath` reflects the module search path and only
+includes path segments appropriate for the version of PowerShell being started.
+
+> [!NOTE]
+> PowerShell sets the value of `$PSHOME\Modules` in the registry. It sets the value of
+> `$HOME\Documents\PowerShell\Modules` each time you start PowerShell.
+
+The default value of Windows PowerShell `$Env:PSModulePath` on Windows is:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\WindowsPowerShell\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+The default value of PowerShell 7 `$Env:PSModulePath` on Windows combines the Windows PowerShell
+paths and the PowerShell 7 paths to provide autoloading of modules:
+
+```powershell
+$nv:PSModulePath -split (';')
+```
+
+```output
+C:\Users\<user>\Documents\PowerShell\Modules
+C:\Program Files\PowerShell\Modules
+C:\Program Files\PowerShell\7\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\WINDOWS\System32\WindowsPowerShell\v1.0\Modules
+```
+
+> [!NOTE]
+> Additional paths may exist if the user has added custom modules or applications that
+> impact the path. You will notice additional paths if a user has modified the path through the
+> registry or **Environment Variables** in **System Settings**.
+
+For more information, see `PSModulePath` in [about_Environment_Variables](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7)
+
+for more information about Modules, see [about_Modules](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/7.0/Microsoft.PowerShell.Core/About/about_Modules.md)
+
+## What Windows PowerShell 5.1 Modules are compatible with PowerShell 7?
+
+Most of the modules you experience in Windows PowerShell 5.1 already work with PowerShell 7, including:
+
+- Azure PowerShell (Az.*)
+- Active Directory
+- Many of the modules in Windows 10 and Windows Server (check with `Get-Module -ListAvailable`)
+
+For the current list of supported modules, see [PowerShell 7 module compatability](/powershell/scripting/whats-new/module-compatibility?view=powershell-7).
+
+[!NOTE]
+> On Windows, we've also added a **UseWindowsPowerShell** switch to Import-Module to ease the
+> transition to PowerShell 7 for those using incompatible modules. This switch creates a proxy
+> module in PowerShell 7 that uses a local Windows PowerShell process to implicitly run any cmdlets
+> contained in that module. For more information on this functionality, check out the Import-Module
+> documentation.
+
+For those modules still incompatible, we're working with a number of teams to add native PowerShell
+7 support, including Microsoft Graph, Office 365, and more.
+
+Azure Cloud Shell has already been updated to use PowerShell 7, and others like the .NET Core SDK
+Docker container images and Azure Functions will be updated soon.
```

Comment:
This doesn't really answer the question posed in the heading. (If I did that in my blog, sorry...) This might even just be a benefit you want to list up top "services like Cloud Shell are already using 7.0, and something something you're gonna want the same version to author with on your dev box"